### PR TITLE
Placement known issue GRC

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/16677

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -634,3 +634,14 @@ You must log out of the external identity provider before you attempt to log out
 //2.4:17229
 
 The backup and restore feature for the hub cluster requires the OpenShift API for Data Protection (OADP) operator. The OADP operator is not available on the IBM Power or IBM Z architectures.
+
```
Comment (gparvin) [inline release_notes/known_issues.adoc]: This content should be under Governance, not under Backup and Restore.

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -634,3 +634,14 @@ You must log out of the external identity provider before you attempt to log out
...
+[#placement-console-support]
+=== Placement resource limitations
+
+As a reminder, a policy must use either a `PlacementRule` or a `Placement` resource to control the deployment of policies to specific managed clusters. If you create policies that use the `Placement` resource, you might encounter the following limitations:
+
+* The placement information is not shown when you view the details of the policy.
```
Comment (gparvin) [inline release_notes/known_issues.adoc]: It's not clear that these limitations apply to the web console and not the cli.  Can that be clarified?

**Review by gparvin — COMMENTED** [review]
I added a couple comments.  Thanks for adding this.

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues.adoc]: Good catch! Thank you !

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (dockerymick) [reply release_notes/known_issues.adoc]: I made the following updates, from: 

* The placement information is not shown when you view the details of the policy.
* The placement information is not removed when the policy is deleted.
* When you edit the policy, the placement details are not updated. 

TO

* The placement information is not shown when you view the details of the policy from the console.
* The placement information is not removed when the policy is deleted from the console.
* When you edit the policy from the console, the placement details are not updated.

**Review by gparvin — APPROVED** [review]

**Comment by dockerymick** [comment]
Thanks for the feedback and review @gparvin !

**Pull request merged**